opennlp-tools/src/main/java/opennlp/tools/cmdline/ArgumentParser.java

```diff
@@ -127,6 +127,21 @@ public Object parseArgument(Method method, String argName, String charsetName) {
     }
   }
 
+  private static class CharacterArgumentFactory implements ArgumentFactory {
+
+    public Object parseArgument(Method method, String argName, String argValue) {
```

Comment:
From the CLI implementers perspective it is very clean. The param.getDefaultEOS() will return a Character.